app/utils/pusher.js

```diff
@@ -28,6 +28,15 @@ TravisPusher.prototype.init = function (config, ajaxService) {
   if (config.host) {
     Pusher.host = config.host;
   }
+
+  if (ENV.pusher.debug) {
+    Pusher.log = function (message) {
+      if (window.console && window.console.log) {
```

Comment:
I think the guard is a bit over the top here. We can expect all browsers these days to just work with 
`console.log();`